OpenSpeedTest-Server.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "OpenSpeedTest-Server",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/openspeedtest/iocage-plugin-OpenSpeedTest-Server.git",
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(80:8080)"
```

Comment:
NGINX is listening to port 8080 in the jail and the FreeNAS host is already listening on port 80 for the UI, so `tcp(8080:8080)` would make more sense.